xml/smb_native_os.xml

```diff
@@ -8,87 +8,108 @@
     <example os.product="Windows NT 4.0">Windows NT 4.0</example>
     <param pos="0" name="os.certainty" value="1.0"/>
     <param pos="0" name="os.vendor" value="Microsoft"/>
+    <param pos="0" name="os.family" value="Windows"/>
     <param pos="1" name="os.product"/>
+    <param pos="0" name="os.device" value="General"/>
```

Comment:
IMO, we should not be sprinkling `General` everywhere, especially if it is just to make the Nexpose UI less ugly in this regard.  IMO, if the banner allows us to determine something less generic, add it as `os.device`, otherwise leave it off.